src/mysql_to_sqlite3/transporter.py

```
@@ -895,34 +895,32 @@ def _create_table(self, table_name: str, attempting_reconnect: bool = False) ->
-
-        If keep_schema is False and schema_name is provided, strip qualifiers like `example`.table.
-        If keep_schema is True, you must ATTACH the SQLite database as that schema name before using the view.
-        """
+    def _mysql_viewdef_to_sqlite(self, view_select_sql: str, view_name: str) -> str:
+        """Convert a MySQL VIEW_DEFINITION (a SELECT ...) to a SQLite CREATE VIEW statement."""
         # Normalize whitespace and avoid double semicolons in output
         cleaned_sql = view_select_sql.strip().rstrip(";")
 
         try:
             tree = parse_one(cleaned_sql, read="mysql")
         except (ParseError, ValueError, Exception):  # pylint: disable=W0718
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)